### PR TITLE
fix(ld-tabs): show tabpanel according to preselected tab

### LINKED ISSUE #179: Controlled Tab component issue.
**Bug description**
When controlling the Tab component through the selected & onTabSelect properties, the initial rendered content is incorrect.

**Steps to reproduce the behavior**
https://codesandbox.io/s/liquid-oxygen-tab-e14r2?file=/src/App.tsx

**Expected behavior**
The selected tab value is "Q1" & therefore "Q1"'s panel content should be displayed.

**Actual behavior**
The selected tab value is "Q1" but it's showing "Q4"'s panel content. Even if we change the selected tab value to "Q2", it always displays "Q4"'s content on the initial render.

### PR DESCRIPTION
**Pull request by borisdiakur**
Fixes #179

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

- [x] unit tests
- [x] tested manually in React app

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing tests pass locally with my changes

### CODE REVIEW

File: src/liquid/components/ld-tabs/ld-tabs.tsx [hunk 1]
```diff
@@ -90,7 +90,10 @@ export class LdTabs {
     let selectedIndex
     this.el.querySelectorAll('ld-tab').forEach((tab, index) => {
       tab.id = `${this.idDescriber}-tab-${index}`
-      if (tab.hasAttribute('selected')) {
+      if (
+        tab.hasAttribute('selected') &&
+        tab.getAttribute('selected') !== 'false'
+      ) {
```
Comment (renet) [inline src/liquid/components/ld-tabs/ld-tabs.tsx:96]:
```suggestion
      if (tab.selected) {
```

**Review by renet — CHANGES_REQUESTED** [review]

File: src/liquid/components/ld-tabs/ld-tabs.tsx (hunk 1 above)
Comment (borisdiakur) [reply src/liquid/components/ld-tabs/ld-tabs.tsx]: Changed ⚡️

**Review by renet — APPROVED** [review]

**Pull request merged**